### PR TITLE
pkp/pkp-lib#10306 Queue jobs unit tests

### PR DESCRIPTION
**Pull request by touhidurabir**
for pkp/pkp-lib#10306

### CODE REVIEW

File: jobs/notifications/IssuePublishedNotifyUsers.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ protected function createMailable(
         Issue $issue,
         User $recipient,
         EmailTemplate $template,
-        Notification $notification
+        Notification|\PKP\notification\Notification $notification
```
Comment (asmecher) [inline jobs/notifications/IssuePublishedNotifyUsers.php:102]: I think there's a missing `use` statement here, not another class named `Notification`.

**Review by asmecher — CHANGES_REQUESTED** [review]
Just one tiny change, thanks!

File: jobs/notifications/IssuePublishedNotifyUsers.php (hunk 1 above)
Comment (touhidurabir) [reply jobs/notifications/IssuePublishedNotifyUsers.php:102]: Kind of but not exactly , we need to have both the `APP\notification\Notification` and `PKP\notification\Notification` as APP Notification has few const and when type hinting in method, we need PKP Notification . updated it .

Comment (asmecher) [reply jobs/notifications/IssuePublishedNotifyUsers.php]: That's better, thanks!

File: classes/decision/types/traits/RequestPayment.php [hunk 2]
```diff
@@ -78,7 +78,6 @@ protected function requestPayment(Submission $submission, User $editor, Context
         $authorIds = $this->getAssignedAuthorIds($submission);
         foreach ($authorIds as $authorId) {
             $notificationMgr->createNotification(
-                Application::get()->getRequest(),
```
Comment (asmecher) [inline classes/decision/types/traits/RequestPayment.php:81]: @touhidurabir, are you sure about this change?

Comment (asmecher) [reply classes/decision/types/traits/RequestPayment.php:81]: Ah, I see this is removed further up the stack because it wasn't used. Never mind.

**Pull request merged**